returntocorp/sgrep-rules
# added debug rule + all cookie rules for laravel

These rules were written for the sake of getting laravel up to tier 1 (specifically for the 'debug code' and 'cookie security' bullet points).

This follows https://cheatsheetseries.owasp.org/cheatsheets/Laravel_Cheat_Sheet.html for best recommendations. 

_To test:_
* run `semgrep --test .` on `php/laravel/security` and check that all tests pass.

_TODO:_
* add tests for default values (ex: catch ` config('app.debug', 'true');`)
